src/commands/commit.ts

```diff
@@ -109,13 +109,14 @@ export async function _commit(
       parsedSettings.blankMigrationContent.trim() + "\n",
     );
   } catch (e) {
-    logDbError(e);
-    // eslint-disable-next-line no-console
-    console.error("ABORTING...");
+    if (parsedSettings.logger instanceof Console) {
```

Comment:
Clarification comment: Since `logDbError` looked like a strictly cli-related helper to print error info, I decided to limit it only to cases when Console is used (default logger value). If a custom logger is provided - it is expected that graphile-migrate is used in library mode, so the error can just be thrown as-is and be captured by an application errors handler, which would most probably log it as well.

Reply:
This comment still applies to newer changes, but instead, I check if logFactory was provided or not. (not sure how to "move" comment from one change snapshot to another :) )